### DISCUSSION TITLE
"Explain Code"  in menu items.

### DISCUSSION THREAD

**Discussion by Yona544** [AliDehbansiahkarbon/ChatGPTWizard#48] [Ideas]
Can you add please 
a. "Explain Code"  
b. "Refactor Code"
in menu items?

**Comment by AliDehbansiahkarbon** [comment]
Sure, 
The "Explain Code" is already added to the editor's context menu, I can add the "Refactor Code" too.
But I don't know if the result of the "Optimize code" prompt differs from the "Refactor code" prompt in ChatGPT's knowledge.
What is your experience with that?

**Reply by Yona544** [reply]
I haven't given "Optimize Code" a try yet. I
I used Refactor with ChatGPT and it was a positive experience.

**Reply by AliDehbansiahkarbon** [reply]
Added.

![image](https://github.com/AliDehbansiahkarbon/ChatGPTWizard/assets/5601608/798b6275-2f13-46d6-9afe-7cbcb99963db)

**Comment by Yona544** [comment]
Thanks 👍

On Thu, Jun 1, 2023 at 10:59 AM Ali Dehbansiahkarbon <
***@***.***> wrote:

> Added.
>
> [image: image]
> <https://user-images.githubusercontent.com/5601608/242637407-798b6275-2f13-46d6-9afe-7cbcb99963db.png>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/AliDehbansiahkarbon/ChatGPTWizard/discussions/48#discussioncomment-6062343>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BAFUORTQAALD76VLLZGELDTXJCU45ANCNFSM6AAAAAAYWEQGDA>
> .
> You are receiving this because you authored the thread.Message ID:
> <AliDehbansiahkarbon/ChatGPTWizard/repo-discussions/48/comments/6062343@
> github.com>
>


-- 
*Yona Tauber | Computer Dimensions*
P 718.625.7043
*F *347.382.9340
E ***@***.***
